.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Please use [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) for private tool artifacts

Reply:
Done